docs/user-guide/rs.rst

```diff
@@ -0,0 +1,230 @@
+.. include:: /content/nemo.rsts
+
+.. _model-aligner-rs:
+
+Model Alignment by Rejection Sampling
+@@@@@@@@@@@@@@@@@@@@@@@
+
+In this tutorial, we will go through the process of aligning a NeMo framework model using rejection sampling. These can be models such as LLaMa2 or Mistral. Our scripts will work the same way.
+
+Rejection Sampling is usually preceded by a Supervised Fine-Tuning (SFT). We should first follow the :ref:`Prerequisite guide <prerequisite>` and the :ref:`SFT guide <sft>`. After obtaining the SFT model, we will also need to train a reward model as in :ref:`PPO guide <ppo>`. We will use the rejection sampling algorithm on the `Anthropic-HH-RLHF <https://huggingface.co/datasets/Anthropic/hh-rlhf>`__ dataset.
+
+Rejection Sampling Training
+############
+
+After you have fine-tuned a GPT model using Supervised Fine-Tuning (SFT), and trained a reward model as explained in the preceding section, you can start aligning the policy using rejection sampling.
+
+During rejection sampling training, we have two models interacting with each other, which Aligner runs in separate jobs:
+
+#. The Policy Network: This is the model we are training, and it should start from an SFT model.
```

Comment:
fix punctuation, remove comma

#. The Policy Network: This is the model we are training and it should start from an SFT model.

Reply:
Fixed the punctuation.